src/App.jsx

```diff
@@ -14,33 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORTER_ALPHABET = 'alphabetically';
+const SORTER_LENGTH = 'length';
+const SORTER_RESET = 'reset';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField }, reversed) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORTER_ALPHABET:
+          return good1.localeCompare(good2);
+        case SORTER_LENGTH:
+          return good1.length - good2.length;
+        case SORTER_RESET:
+          return [...goodsFromServer];
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (reversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
+
+  const visibleGoods = getPreparedGoods(
+    goodsFromServer,
+    { sortField },
+    reversed,
+  );
+
+  function isNotClicked(sorter) {
+    if (sortField !== sorter) {
+      return true;
+    }
+
+    return false;
```

Comment:
```suggestion
    return sortField !== sorter;
```

We can simplify it a bit